staging/src/k8s.io/apiserver/plugin/pkg/authorizer/webhook/webhook.go

```diff
@@ -199,12 +218,55 @@ func (w *WebhookAuthorizer) Authorize(attr authorizer.Attributes) (authorized bo
 
 //TODO: need to finish the method to get the rules when using webhook mode
 func (w *WebhookAuthorizer) RulesFor(user user.Info, namespace string) ([]authorizer.ResourceRuleInfo, []authorizer.NonResourceRuleInfo, bool, error) {
+	if w.subjectRulesReview == nil {
+		incomplete := true
+		return nil, nil, incomplete, fmt.Errorf("webhook authorizer does not support user rule resolution")
+	}
+
+	r := &authorization.SubjectRulesReview{
+		Spec: authorization.SubjectRulesReviewSpec{
+			Namespace: namespace,
+			User:      user.GetName(),
+			UID:       user.GetUID(),
+			Groups:    user.GetGroups(),
+			Extra:     convertToSARExtra(user.GetExtra()),
+		},
+	}
+
 	var (
-		resourceRules    []authorizer.ResourceRuleInfo
-		nonResourceRules []authorizer.NonResourceRuleInfo
+		result *authorization.SubjectRulesReview
+		err    error
 	)
-	incomplete := true
-	return resourceRules, nonResourceRules, incomplete, fmt.Errorf("webhook authorizer does not support user rule resolution")
+	webhook.WithExponentialBackoff(w.initialBackoff, func() error {
```

Comment:
Don't know if we want to cache results here as well cc @cjcullen

Reply:
I'm not as concerned about caching here. This isn't in the apiserving path of every request like the SubjectAccessReview webhook.